dataset_interfaces/factory.py

```diff
@@ -1,24 +1,24 @@
 import os
 
 from dataset_interfaces.interface import TestExample, DatasetInterface
-from datasets.conficting_personal_information import ConflictingPersonalInformationDataset
-from datasets.delayed_recall import DelayedRecallDataset
-from datasets.how_to_think import HowToThinkDataset
-from datasets.instruction_recall import InstructionRecallDataset
-from datasets.prospective_memory import ProspectiveMemoryDataset
-from datasets.colours import ColourDataset
-from datasets.jokes import JokesDataset
-from datasets.locations import LocationsDataset
-from datasets.locations_directions import LocationsDirectionsDataset
-from datasets.name import NamesDataset
-from datasets.name_list import NameListDataset
-from datasets.sally_ann import SallyAnneDataset
-from datasets.shopping import ShoppingDataset
-from datasets.spy_meeting import SpyMeetingDataset
-from datasets.trigger_response import TriggerResponseDataset
-from datasets.kv import KVPairsDataset
-from datasets.chapterbreak import ChapterBreakDataset
-from datasets.restaurant import RestaurantDataset
+from local_datasets.conficting_personal_information import ConflictingPersonalInformationDataset
```

Comment:
if reasonable, it would be good to not change this, and so also keep the num. of changes low.

Reply:
Right oh I will revert this. That will remove a lot of the changes.